SU2_CFD/include/fluid/CFluidScalar.hpp

```diff
@@ -0,0 +1,69 @@
+#pragma once
+
+#include <vector>
+#include <memory>
+
+#include "CFluidModel.hpp"
+
+class CFluidScalar : public CFluidModel {
+
+protected:
+  unsigned short n_species_mixture = 0.0;
+  // su2double viscosityMixture = 0.0;
+  // su2double Density;
+  // su2double Temperature;
+  su2double Gas_Constant = 0.0; 
+  su2double Gamma = 0.0; 
+  
+  bool wilke;
+  bool davidson; 
+
+  std::vector<su2double> massFractions; 
+  std::vector<su2double> moleFractions;  
+  std::vector<su2double> molarMasses;
+  std::vector<su2double> laminarViscosity;
+  std::vector<su2double> specificHeat; 
+  // std::vector<su2double> laminarViscosity = {1.099e-05, 1.823e-05}; 
+  // std::vector<su2double> laminarViscosity = {1.126e-05, 1.849e-05}; //corresponds with Fluent 
+  std::vector<su2double> laminarthermalConductivity; 
+  
+  unique_ptr<CViscosityModel> LaminarViscosityPointers[100];  
+  unique_ptr<CConductivityModel> ThermalConductivityPointers[100]; 
```

Comment:
Yes I hardcoded this because I don't know how to set the size corresponding to the number of species in the mixture. 
Right now, I changed it to e.g.:
std::vector<std::unique_ptr<CViscosityModel>> LaminarViscosityPointers;

and kept void CFluidScalar::SetLaminarViscosityModel(const CConfig* config) {} in CFluidScalar.cpp the same. Is this the correct way?